alerts/README.md

```diff
@@ -2,15 +2,58 @@
 title: Alerts
 ---
 
-This plugin extension contains [Prometheus Alertmanager](https://github.com/prometheus/alertmanager) via [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator) and [Supernova](https://github.com/sapcc/supernova), the holistic alert management UI.
+Learn more about the **alerts** plugin. Use it to activate Prometheus alert management for your Greenhouse organisation. 
 
-# Owner
+The main terminologies used in this document can be found in [core-concepts](https://cloudoperators.github.io/greenhouse/docs/getting-started/core-concepts).
 
-1. Richard Tief (@richardtief)
-2. Tommy Sauer (@viennaa)
-3. Martin Vossen (@artherd42)
 
-### alerts alertmanager parameters
+## Overview
+
+This Plugin includes a preconfigured _Prometheus Alertmanager_, which is deployed and managed via the _Prometheus_ Operator, and _Supernova_, an advanced user interface for Prometheus Alertmanager. In addition, certificates are automatically generated to ensure _Prometheus_ authentication sending alerts to the Alertmanager as well as Slack notification templates are provided.
+
+Components included in this Plugin:
+
+- [Prometheus Alertmanager](https://prometheus.io/docs/alerting/alertmanager)
+- [Supernova](https://github.com/cloudoperators/juno/tree/main/apps/supernova)
+
+Optionally, [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) can be used to manage _Prometheus_ and _Alertmanager_ instances. Usually, **kube-monitoring** Plugin installs _Prometheus Operator_.
```

Comment:
```suggestion
Optionally, [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) can be used to manage _Prometheus_ and _Alertmanager_ instances. Usually, **kube-monitoring** Plugin installs _Prometheus Operator_.
```

The config says this installs Prometheus Operator:
https://github.com/cloudoperators/greenhouse-extensions/blob/main/alerts/charts/values.yaml#L104-L106

Should we put in an either or? Either we have kube-monitoring installing this, or - in the unlikely case of not having kube-monitoring this option should be set. 

What happens if we deploy both with operator enabled?

Reply:
It is disabled again in the alerts `plugindefinition.yaml` as we intend to install it with kube-monitoring. We currently have no solution to this problem other than this documentation. There is a issue in our backlog to track - https://github.com/cloudoperators/greenhouse-extensions/issues/140

Reply:
Ok, rewriting it.

```suggestion

This Plugin usually is deployed along the **kube-monitoring** Plugin and does not deploy the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) itself. However, if you are intending to use it stand-alone, you need to explicitly enable the deployment of Prometheus Operator, otherwise it will not work. It can be done in the configuration interface of the plugin.

Reply:
got lost, again here:

```suggestion
This Plugin usually is deployed along the **kube-monitoring** Plugin and does not deploy the [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) itself. However, if you are intending to use it stand-alone, you need to explicitly enable the deployment of Prometheus Operator, otherwise it will not work. It can be done in the configuration interface of the plugin.
```